app/retail/emails.py

```diff
@@ -443,7 +443,7 @@ def render_admin_contact_funder(bounty, text, from_user):
     return response_html, response_txt
 
 
-def render_funder_stale(github_username, days=30, time_as_str='about a month'):
+def render_funder_stale(github_username, days=60, time_as_str='a couple months'):
```

Comment:
will need to update this line in the crontab too.  this change only edits the default arguments, not the actually executed ones

```
15 10 * * * cd gitcoin/coin; bash scripts/run_management_command_if_not_already_running.bash funder_stale_email 30  >> /var/log/gitcoin/funder_stale_email.log  2>&1
```

Reply:
taken care